gradle.properties

```diff
@@ -33,6 +33,8 @@ POM_LICENCE_DIST=repo
 POM_DEVELOPER_ID=gcacace
 POM_DEVELOPER_NAME=Gianluca Cacace
 
-ANDROID_BUILD_TARGET_SDK_VERSION=25
-ANDROID_BUILD_TOOLS_VERSION=25.0.3
```

Comment:
Why remove this? Newest one should be 29.0.3

Reply:
Yes, I removed it as gradle seems to use the proper build tools version without specifying, but if it's better, we can specify 29.0.3

Reply:
Not sure, they mention the minimum required, but don't say if that means the latest one with latest bug fixes. As you prefer.

Reply:
I guess both build tools have bugs 😅 so let's go with the latest ones. Switching to 29.0.3